src/Merchants/Content/Merchant/MerchantEntity.php

```diff
@@ -135,6 +136,11 @@ class MerchantEntity extends Entity
      */
     protected $orders;
 
+    /**
+     * @var MediaCollection
```

Comment:
Can be null

Reply:
Ah, forgot that collections are not just instantiated emptily....